lexos/static/js/scripts_cut.js

```diff
@@ -1,382 +1,663 @@
+            <input type="text" class="indivMSinput" \
+            name="MScutWord_' + fileID + '" id="MScutWord' + fileID + '" \
+            value="" \
+            style="margin-left:3px;width:130px;"/></span></div>\
+            </div></fieldset></div>'
+}
+
+// process
+/**
+ * checks the form data for errors and warnings
+ * @param {string} action
+ */
 function process (action) {
-  $('#status-prepare').css({ 'visibility': 'visible', 'z-index': '400000' })
-  $('#formAction').val(action)
-  $.when(checkForErrors()).done(function () {
-    if ($('#hasErrors').val() == 'false') {
-      checkForWarnings()
-      $.when(checkForWarnings()).done(function () {
-        if ($('#needsWarning').val() == 'false') {
-          doAjax(action)
+    $('#status-prepare').css({'visibility': 'visible', 'z-index': '400000'})
+    $('#formAction').val(action)
+    $.when(checkForErrors()).done(function () {
+        if ($('#hasErrors').val() === 'false') {
+            checkForWarnings()
+            $.when(checkForWarnings()).done(function () {
+                if ($('#needsWarning').val() === 'false') {
+                    doAjax(action)
+                }
+            })
         }
-      })
-    }
-  })
-}
+    })
+} // end process
 
+//==Warning Modal Click Functions=================
 // Handle the Continue button in the warning modal
-$(document).on('click', '#warningContinue', function (event) {
-  $('#needsWarning').val('false')
-  action = $('#formAction').val()
-  $('#warning-modal').modal('hide')
-  doAjax(action)
-  $('#status-prepare').css({ 'visibility': 'visible', 'z-index': '400000' })
+$(document).on('click', '#warningContinue', function () {
+    $('#needsWarning').val('false')
+    let action = $('#formAction').val()
+    $('#warning-modal').modal('hide')
+    doAjax(action)
+    $('#status-prepare').css({'visibility': 'visible', 'z-index': '400000'})
 })
 
 // Handle the Timer Cancel button in the warning modal
-$(document).on('click', '#timerCancel', function (event) {
-  $('#needsWarning').val('false')
-  $('#hasErrors').val('false')
-  xhr.abort()
-  $('#warning-modal-footer').append('<button>Moo</button>')
-  $('#warning-modal').modal('hide')
-  $('#status-prepare').css('visibility', 'hidden')
+$(document).on('click', '#timerCancel', function () {
+    $('#needsWarning').val('false')
+    $('#hasErrors').val('false')
+    xhr.abort()
+    $('#warning-modal-footer').append('<button>Moo</button>')
+    $('#warning-modal').modal('hide')
+    $('#status-prepare').css('visibility', 'hidden')
 })
+//=====================================================
 
-// Function to convert the form data into a JSON object
+
+// jsonifyForm
+/**
+ *  converts the form data into a JSON object
+ */
 function jsonifyForm () {
-  var form = {}
-  $.each($('form').serializeArray(), function (i, field) {
-    form[field.name] = field.value || ''
-  })
-  return form
-}
+    let form = {}
+    $.each($('form').serializeArray(), function (i, field) {
```

Comment:
I think we can use map here. Please investigate. also move this function outside of the `$()`, and document the return.